components/Dialogs/examples/supplemental/DialogsTypicalUseSupplemental.m

```diff
@@ -110,7 +111,8 @@ - (CGSize)preferredContentSize {
 }
 
 - (IBAction)dismiss:(id)sender {
-  [self.presentingViewController dismissViewControllerAnimated:YES completion:NULL];
+  NSURL *testURL = [NSURL URLWithString:@"https://www.material.io"];
+  [[UIApplication mdc_safeSharedApplication] performSelector:@selector(openURL:) withObject:testURL];
```

Comment:
I don't know much about mdc_safeSharedApplication. Is this so it's safe in an extension?

Reply:
Yeah thats correct. In extensions you can't get access to [UIApplication sharedApplication], so `mdc_safeSharedApplication` returns nil inside extensions.

Reply:
In that case, don't hate me, but a tiny comment explaining that would be helpful since it's educational code.